distiller/models/imagenet/mobilenetv3_duoli.py

```diff
@@ -257,8 +258,13 @@ def _initialize_weights(self):
                 m.bias.data.zero_()
 
 
-def mobilenetv3_duoli(pretrained=False, mobilenet_early_exit_branch=None, **kwargs):
+def mobilenetv3_duoli(pretrained=False, mobilenet_early_exit_branch=None, pretrained_checkpoint_path=None, **kwargs):
+    msglogger = logging.getLogger()
```

Comment:
what is the default name of the logger if not specified? I am used to seeing logging.getLogger(__name__).

Reply:
From the documentation
> Return a logger with the specified name or, if name is None, return a logger which is the root logger of the hierarchy.